block_chain_test.py

```diff
@@ -25,20 +27,51 @@ def test_block_chain(self):
         host1 = config1["dbHost"]
         host2 = config2["dbHost"]
         ts_count = 30
-        self.data = utils.login(config1["url"], config1['private_key'])
+        self.data1 = utils.login(config1["url"], config1['private_key'])
         i = 1
         while i < ts_count:
             contName = self.create_contract(config1["url"], config1['private_key'])
             i = i + 1
             time.sleep(1)
         time.sleep(15)
-        self.assertEqual(utils.get_count_records_block_chain(host1, db1, login1, pas1), 30, "There isn't 30 records in block_chain1")
-        self.assertEqual(utils.get_count_records_block_chain(host2, db2, login2, pas2), 30, "There isn't 30 records in block_chain2")
         count_contracts1 = utils.getCountDBObjects(host1, db1, login1, pas1)["contracts"]
         count_contracts2 = utils.getCountDBObjects(host2, db2, login2, pas2)["contracts"]
-        self.assertTrue(utils.compare_node_positions(host1, db1, login1, pas1), "Incorrect order of nodes in block_chain")
-        self.assertEqual(count_contracts1, count_contracts2,"Different count")
-        self.assertTrue(utils.compare_keys_cout(host1, db2, login1, pas1), "There are different count of keys in block_chain")
+        fail = 0
+        msg = ""
+        if(count_contracts1 != count_contracts2):
```

Comment:
self.assertEqual(count_contracts1, count_contracts2, "Contract count is different")